go/vt/vttablet/onlineddl/executor.go

```diff
@@ -4693,6 +4693,31 @@ func (e *Executor) SubmitMigration(
 	return result, nil
 }
 
+// ShowMigrations shows migrations, optionally filtered by a condition
+func (e *Executor) ShowMigrations(ctx context.Context, show *sqlparser.Show) (result *sqltypes.Result, err error) {
+	if atomic.LoadInt64(&e.isOpen) == 0 {
+		return nil, vterrors.New(vtrpcpb.Code_FAILED_PRECONDITION, "online ddl is disabled")
```

Comment:
Nit, but IMO `OnlineDDL` or `online DDL` is better. I feel like perhaps the former so as to identify that it's really the Vitess OnlineDDL functionality that's disabled (MySQL itself supports online DDL, and this could be confusing for MySQL users).

Reply:
done